src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -587,7 +592,7 @@ export default {
 
 	.preview {
 		border-radius: var(--border-radius);
-		max-width: 100%;
+		max-width: 600px;
```

Comment:
Enough for standing out images:
```suggestion
		max-width: min(100%, 600px);
```